### PR TITLE
update zscaler-web-security: Zscaler pricing info

### PR DESCRIPTION
**Pull request by rikishi-c**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/security-threat-detection/zscaler-web-security.md [hunk 1]
```diff
@@ -46,6 +46,11 @@ Zscaler offers a virtual appliance, called Nanolog Streaming Service (NSS) to st
 
 To stream logs to the Sumo Logic Syslog Source, perform steps A, B, and C detailed in the “NSS Configuration Guide” at: [https://support.zscaler.com/hc/en-us...guration-Guide](https://support.zscaler.com/hc/en-us/articles/205518405-NSS-Configuration-Guide).
 
+:::note
+Zscaler Log Streaming is an add on for customers that do not have it as part of an existing bundle. Please check your subscription and work with your Zscaler support team to see if CloudNSS is included in your bundle.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/zscaler-web-security.md:50]:
```suggestion
Zscaler Log Streaming is an add-on for customers that do not have it as part of an existing bundle. Please check your subscription and work with your Zscaler support team to see if CloudNSS is included in your bundle.
```

**Comment by kimsauce** [comment]
Hi @rikishi-c , thank you for your submission! Can you please sign our CLA (linked in comments above)? It's required for all internal and external contributors.

File: docs/integrations/security-threat-detection/zscaler-web-security.md [hunk 2]
```diff
@@ -46,6 +46,11 @@ Zscaler offers a virtual appliance, called Nanolog Streaming Service (NSS) to st
 
 To stream logs to the Sumo Logic Syslog Source, perform steps A, B, and C detailed in the “NSS Configuration Guide” at: [https://support.zscaler.com/hc/en-us...guration-Guide](https://support.zscaler.com/hc/en-us/articles/205518405-NSS-Configuration-Guide).
 
+:::note
+Zscaler Log Streaming is an add-on for customers that do not have it as part of an existing bundle. Please check your subscription and work with your Zscaler support team to see if CloudNSS is included in your bundle.
+
+More information can be found at https://www.zscaler.com/pricing-and-plans
```
[the remark this answers is not in the document]
Comment (rikishi-c) [reply docs/integrations/security-threat-detection/zscaler-web-security.md:52]: Thanks

**Comment by rikishi-c** [comment]
> Hi @rikishi-c , thank you for your submission! Can you please sign our CLA (linked in comments above)? It's required for all internal and external contributors.

Done!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**